rwkv/chat_with_bot.py

```diff
@@ -4,24 +4,92 @@
 import sys
 import argparse
 import pathlib
+import copy
 import sampling
 import tokenizers
 import rwkv_cpp_model
 import rwkv_cpp_shared_library
 
 # ======================================== Script settings ========================================
 
-# Copied from https://github.com/ggerganov/llama.cpp/blob/6e7801d08d81c931a5427bae46f00763e993f54a/prompts/chat-with-bob.txt
-prompt: str = """Transcript of a dialog, where the User interacts with an Assistant named Bob. Bob is helpful, kind, honest, good at writing, and never fails to answer the User's requests immediately and with precision.
+# Copied from https://github.com/BlinkDL/ChatRWKV/blob/9ca4cdba90efaee25cfec21a0bae72cbd48d8acd/chat.py#L92-L178
+CHAT_LANG = 'English' # English // Chinese // more to come
 
-User: Hello, Bob.
-Bob: Hello. How may I help you today?
-User: Please tell me the largest city in Europe.
-Bob: Sure. The largest city in Europe is Moscow, the capital of Russia."""
+QA_PROMPT = False # True: Q & A prompt // False: User & Bot prompt
+# 中文问答设置QA_PROMPT=True（只能问答，问答效果更好，但不能闲聊） 中文聊天设置QA_PROMPT=False（可以闲聊，但需要大模型才适合闲聊）
```

Comment:
Looks like there is way more content in the Chinese comment. Can it be also translated to English?
(also, I suggest to move the comment before variable declaration)

Reply:
OK, I refined this comment.